03.asynchronous/callback/callback.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+import sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(":memory:");
+
+db.run(
+  "CREATE TABLE books(id INTEGER PRIMARY KEY AUTOINCREMENT,title TEXT NOT NULL)",
```

Comment:
SQL のスタイルが整っていなさそうです。
```suggestion
  "CREATE TABLE books(id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL)",
```